### PR TITLE
Created password change successful page

### LINKED ISSUE #217: Password change successful page
### Summary and Background
Password change successful page for super admin, admins, and volunteers.

### User Story
As a Super Admin/Admin/Volunteer, I want confirmation if my password change was successful.

### Solution (Core Elements)
Create password change successful page.

User sees this page if their password change (#216) was successful.

User can click 'Login' to go to the login page #214.

### Mockups

Other designs: 
https://www.figma.com/file/tOK6AjkaRZ1NBXo191D64C/Sistering-%2F-User-Authentication-Process?node-id=390%3A1975

### Technical
Create a new component in `frontend\src\components\auth` called `PasswordResetSuccess.tsx` and create a new route called `/password-reset-success`. Add it to `App.tsx`.

### PR DESCRIPTION
**Pull request by eevanwong**
## Ticket link

Closes #217 

## Implementation description
* Created page, added necessary routing

## Steps to test
1. go to http://localhost:3000/password-reset-success

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by eevanwong** [comment]
image

File: frontend/src/components/auth/PasswordResetSuccess.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <Box display="flex">
+      <Box margin="48px auto" textAlign="center">
+        <Logo />
+        <Text fontSize="xl" fontWeight="bold" marginBottom="24px">
+          Password Change Successful!
+        </Text>{" "}
+        <Text fontSize="l" marginBottom="24px">
+          You may now login with your new credentials.
+        </Text>{" "}
```
Comment (MatoPlus) [inline frontend/src/components/auth/PasswordResetSuccess.tsx:18]:
```suggestion
        <Text fontSize="14px" marginBottom="24px">
          You may now login with your new credentials.
        </Text>
```
Looking at the figma file, I think the text size should be font 14px unless this is inconsistent with our other pages eg: account successfullly created.

File: frontend/src/components/auth/PasswordResetSuccess.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const PasswordResetSuccessPage = (): React.ReactElement => {
+  const history = useHistory();
+
+  return (
+    <Box display="flex">
+      <Box margin="48px auto" textAlign="center">
+        <Logo />
+        <Text fontSize="xl" fontWeight="bold" marginBottom="24px">
+          Password Change Successful!
+        </Text>{" "}
```
Comment (MatoPlus) [inline frontend/src/components/auth/PasswordResetSuccess.tsx:15]:
```suggestion
        </Text>
```
extra space?

**Review by MatoPlus — DISMISSED** [review]
LGTM with suggestions/nits

**Review by MatoPlus — APPROVED** [review]
Thanks for the clean up : )

**Pull request merged**